### PR TITLE
Database edit

### PR DESCRIPTION
**Pull request by Sindahlof**
Added editing for meal items, no deleteing individual meal items or entire meals

### CODE REVIEW

File: CarboPlanner/src/components/VerticalLine.tsx
Comment (frmad) [inline CarboPlanner/src/components/VerticalLine.tsx:1]: is it vertical or horizontal?

File: CarboPlanner/src/views/ResultScreen.tsx
Comment (frmad) [inline CarboPlanner/src/views/ResultScreen.tsx:1]: Maybe it should be split into different components

**Review by frmad — CHANGES_REQUESTED** [review]

File: CarboPlanner/src/components/VerticalLine.tsx
Comment (Sindahlof) [reply CarboPlanner/src/components/VerticalLine.tsx:1]: I have not changed this

File: CarboPlanner/src/views/ResultScreen.tsx
Comment (Sindahlof) [reply CarboPlanner/src/views/ResultScreen.tsx:1]: Then we would have to build result screen again

**Comment by frmad** [comment]
fine

**Review by frmad — APPROVED** [review]

**Pull request merged**